src/nvim/highlight_group.c

```diff
@@ -264,6 +264,18 @@ static const char *highlight_init_both[] = {
   "default link @preproc PreProc",
   "default link @debug Debug",
   "default link @tag Tag",
+
+  // LSP semantic tokens
+  "default link @class Type",
+  "default link @struct Type",
+  "default link @enum Type",
+  "default link @enumMember Constant",
+  "default link @event Identifier",
+  "default link @interface Identifier",
+  "default link @modifier Identifier",
+  "default link @regexp SpecialChar",
+  "default link @typeParameter Type",
+  "default link @decorator Identifier",
```

Comment:
@clason are these default links okay?

Reply:
Looks reasonable enough to me, but I personally don't rely on the default links, so no strong feelings either way. @gpanders?

(Before 0.9, we probably want to take a good hard look at the default links (or colors) for the new treesitter and lsp groups anyway.)

Reply:
Meaning, it shouldn't block this. As long as the links are there, we can always tweak them later (and probably will have to when modifiers enter the mix).

Oh, but:

> "default link @struct Type",

I'd prefer to rename the already existing `@structure` group instead -- having both is needlessly confusing.